src/DynamoSandbox/SplashScreen.xaml.cs

```diff
@@ -111,6 +111,7 @@ internal async void SetLabels()
             await webView.CoreWebView2.ExecuteScriptAsync("window.setLabels({" +
                $"welcomeToDynamoTitle: '{Properties.Resources.SplashScreenWelcomeToDynamo}'," +
                $"launchTitle: '{Properties.Resources.SplashScreenLaunchTitle}'," +
+               $"importSettingsTitle: 'dd{Properties.Resources.SplashScreenImportSettings}'," +
```

Comment:
typo here `dd`

Reply:
Ahh.. I was testing if the string does update from Dynamo to splash screen. Looks like this was left out, Will submit a fix. Sorry

Reply:
https://github.com/DynamoDS/Dynamo/pull/13466